package.json

```diff
@@ -11,9 +11,12 @@
     "i18next-browser-languagedetector": "^5.0.0",
     "i18next-http-backend": "^1.0.17",
     "mdbreact": "^4.27.0",
+    "node-sass": "^4.14.1",
```

Comment:
are you using this dependence? if not please remove it

Reply:
yes i use it , the scss won't work if i didn't apply this